app/main.py

```diff
@@ -1,8 +1,40 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str) -> None:
+        self.comform_class = comfort_class
```

Comment:
There is a typo in the attribute name `comform_class`. It should be `comfort_class` to match the constructor parameter and the task description.